src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -81,15 +81,30 @@ public InfrastrukturabfrageModel saveInfrastrukturabfrage(final Infrastrukturabf
     }
 
     /**
-     * Diese Methode updated ein {@link InfrastrukturabfrageModel}.
+     * Diese Methode updated ein {@link InfrastrukturabfrageModel} ausgenommen vom Abfrage Status.
      *
      * @param abfrage zum Updaten
      * @return das geupdatete {@link InfrastrukturabfrageModel}
      * @throws EntityNotFoundException falls die Abfrage identifiziert durch die {@link InfrastrukturabfrageModel#getId()} nicht gefunden wird
      * @throws UniqueViolationException falls der Name der Abfrage {@link InfrastrukturabfrageModel#getAbfrage().getNameAbfrage} ()} bereits vorhanden ist
      * @throws OptimisticLockingException falls in der Anwendung bereits eine neuere Version der Entität gespeichert ist
      */
-    public InfrastrukturabfrageModel updateInfrastrukturabfrage(final InfrastrukturabfrageModel abfrage) throws EntityNotFoundException, UniqueViolationException, OptimisticLockingException {
+    public InfrastrukturabfrageModel updateInfrastrukturabfrageWithoutStatus(final InfrastrukturabfrageModel abfrage) throws EntityNotFoundException, UniqueViolationException, OptimisticLockingException {
+        InfrastrukturabfrageModel abfragedb = this.getInfrastrukturabfrageById(abfrage.getId());
```

Comment:
Die Lösung ist richtig gut. Einfach und sehr effektiv! :-)

Attribut "abfragedb" final und in Camelcase. -> abfrageDb